resources/migrations/000_migrations.yaml

```diff
@@ -16054,7 +16054,7 @@ databaseChangeLog:
       rollback: # no change
 
   - changeSet:
-      id: v48.00-053
+      id: v48.00-052
```

Comment:
This will break the app DB for anyone who's already ran the previous version of the 053 migration. Not the end of the world but might be a bit nicer to keep 053 as it is and either use 052 or 054 (there's no problem with skipping migration IDs)

Reply:
Good idea, I'll do that.

Reply:
Forgot about the Liquibase checksums.